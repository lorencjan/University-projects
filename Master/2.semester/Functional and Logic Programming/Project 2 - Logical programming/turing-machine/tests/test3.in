S   A R
A a A b
A b A R
A   F  
 aaaaaa
