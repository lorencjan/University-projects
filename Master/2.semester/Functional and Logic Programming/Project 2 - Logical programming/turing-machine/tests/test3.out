S aaaaaa
 Aaaaaaa
 Abaaaaa
 bAaaaaa
 bAbaaaa
 bbAaaaa
 bbAbaaa
 bbbAaaa
 bbbAbaa
 bbbbAaa
 bbbbAba
 bbbbbAa
 bbbbbAb
 bbbbbbA
 bbbbbbF 
